Dockerfile

```diff
@@ -2,7 +2,7 @@
 FROM rust:1.25.0-stretch as builder
 ADD . /app
 WORKDIR /app
-ARG RUST_TOOLCHAIN=nightly-2018-04-29
+ARG RUST_TOOLCHAIN=nightly
```

Comment:
Is this going to respect the locked one you referenced above?

Reply:
I only required that locked version with previous versions of rust nightly which caused diesel to fail to compile. Current versions of nightly are no longer a problem, thus I can remove the date requirement.